.github/workflows/cross-version-tests.yml

```
@@ -60,8 +60,24 @@ jobs:
         run: |
           conda install python=3.6
           python --version
+      - name: Get cache key
+        env:
+          INSTALL_COMMAND: ${{ matrix.install }}
+        id: get-cache-key
+        run: |
+          date=$(/bin/date -u "+%Y%m%d")
+          hash=$(echo -n "$INSTALL_COMMAND" | sha256sum)
+          echo "::set-output name=key::$date-$hash"
+      - uses: actions/cache@v2
+        if: ${{ matrix.job_name == 'spark / dev / autologging' }}
```

I'll fix this line to:

```suggestion
        if: ${{ matrix.package == 'pyspark' && matrix.version == 'dev' }}
```

Once #4284 has been merged.